lib/server/services/run/RunService.js

```diff
@@ -331,14 +331,15 @@ const updateEorReasonsOnRun = async (runId, userName, eorReasonPatches) => {
 /**
  * Method to process a user for the run
  *
- * @param {number} externalId - external identifier of the user
- * @param {string|null} name - name of the user or empty
+ * @param {User|null} userData - user object, either userO2Start or userO2Stop
```

Comment:
It is not really a user, but a user identification/creation paylaod. You should type it as `object|null` then specify its properties (as you already done)

Reply:
I understand, I changed it now